tests/features/GPU_TESTING.md

```
@@ -0,0 +1,183 @@
+
+```bash
+oc delete localqueue test-local-queue cpu-local-queue -n ${X_TENANT}
+oc delete clusterqueue gpu-cluster-queue cpu-only-cluster-queue
+oc delete resourceflavor gpu-a100 gpu-v100 default-flavor
+```
+
+## Manual/Standalone Testing
+
+The standalone test script deploys the feature images, sets up resources, runs all scenarios, and cleans up.
+
+### Prerequisites
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)